e2e/tests/integration/cedat-arrival-export.ts

```diff
@@ -1,118 +0,0 @@
-import {todayAtUtc, todayAtUtcString} from "../support/time-helpers";
-import {manifestForDateTime, passengerList} from "../support/manifest-helpers";
-import {paxRagGreenSelector} from "../support/commands";
-
-
-describe("CEDAT arrival exports", () => {
```

Comment:
Do we not need this test any more ?

Reply:
Cedat said they don't use DRT anymore so we don't need to retain a separate export for them. In fact we need a ticket to remove the cedat specific export code!